src/jshint.js

```
@@ -1265,7 +1264,7 @@ var JSHINT = (function() {
             warning("W121", left, nativeObject);
         }
 
-        if (left.identifier) {
+        if (left.identifier && !left.isMetaProperty) {
```

can we throw an error if it is assigned to?